Common/src/main/java/jeresources/jei/mob/MobWrapper.java

```diff
@@ -151,4 +151,12 @@ private int getOffsetY(LivingEntity livingEntity) {
         else if (livingEntity instanceof AbstractPiglin) offsetY = -15;
         return offsetY;
     }
-}
+    
+    public boolean hasSpawnEgg() {
+        return getSpawnEgg() != null ? true : false;
```

Comment:
This is a bit redundant :D, it can just be `return getSpawnEgg() != null`

Reply:
Noted :)

Reply:
noted :)